### PR TITLE
Remove util subdirectory. Add exist call. Add percentage disable on webu...

### PR DESCRIPTION
**Pull request by sqrl**
...i. Fix some style.

### CODE REVIEW

File: packrat/packrat.py [hunk 1]
```diff
@@ -26,6 +26,17 @@ def set_or_get(key=None):
     return send_file(file[0], as_attachment=True, attachment_filename=file[1])
 
 
+@app.route('/exists/<key>', methods=['GET'])
```
Comment (kevinlaube) [inline packrat/packrat.py:29]: Although this will certainly work well given that we are hashing keys before feeding them to packrat, would this be a good time to update the routes we are using to that nothing exists straight off the root? (ie /exists potentially conflicts with key=exists) Maybe if we did something like changing the keys route to '/files/<key>' ?

**Comment by kevinlaube** [comment]
Looks good to me. I had another thought we can discuss in person, but what do you think about adding a backdoor clear method? Something like the route /clear will reset the cache without requiring us to knock down the server. It might be useful if we ever end up with a series of malformed tracefiles in the cache. We could even create a few binaries to specifically call the clear command and save them in the workspace in case the need arises.

File: packrat/packrat.py (hunk 1 above)
Comment (sqrl) [inline packrat/packrat.py:29]: It's a little confusing but not too bad: You can't mask the exist endpoint with uploading a key of 'exists':

127.0.0.1/exists - still looks for an item with key 'exists'
127.0.0.1/exists/ - still looks for an item with key 'exists'
127.0.0.1/exists/a - checks for the existence of a key named 'a'
127.0.0.1/exists/exists - checks for the existence of a key named 'exists'

Comment (sqrl) [inline packrat/packrat.py:29]: I'm fine with this change, though. Can it be a separate commit?

**Comment by sqrl** [comment]
Sounds good to me. We can add that in another change. Can you +1 this?

**Comment by kevinlaube** [comment]
+1
